applications/StructuralMechanicsApplication/custom_utilities/shell_cross_section.cpp

```diff
@@ -1173,7 +1173,7 @@ void ShellCrossSection::CalculateIntegrationPointResponse(IntegrationPoint& rPoi
     rVariables.DeterminantF0 = 1.0;
 
     // calculate the material response
-
+    rMaterialValues.GetOptions().Set(ConstitutiveLaw::USE_ELEMENT_PROVIDED_STRAIN, true); // based on suggestion of Riccardo in #1545

```

Comment:
I am  not very favorable for this kind of commentaries that will lost context ten minutes after merge, please remove.

Reply:
ok